src/hazelweb/gui/Cell.re

```diff
@@ -56,6 +56,22 @@ let focus = () => {
   JSUtil.force_get_elem_by_id("cell")##focus;
 };
 
+let caret_from_pos = (x: float, y: float): Vdom.Node.t => {
```

Comment:
Let's move this caret positioning logic into `UHCode` (where `UHCode.view` additionally take the `(row, col)` as an argument), reason being we've done that in the `livelits` branch where you have nested splices and cursor maps inside of embedded guis so eventually it will need to move there anyway.